201-decrypt-running-windows-vm/azuredeploy.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the virtual machine"
+      }
+    },
+    "volumeType": {
+      "type": "string",
+      "defaultValue": "All",
+      "allowedValues": [
+        "All",
+        "OS",
+        "Data"
+      ],
+      "metadata": {
+        "description": "Type of the volume OS or Data or All to perform decryption operation"
+      }
+    },
+    "sequenceVersion": {
+      "type": "string",
+      "defaultValue": "1.0",
+      "metadata": {
+        "description": "Pass in an unique value like a GUID everytime the operation needs to be force run"
+      }
+    }
+  },
+  "variables": {
+    "extensionName": "AzureDiskEncryption",
+    "extensionVersion": "1.1",
+    "encryptionOperation": "DisableEncryption",
+    "updateEncryptionSettingsUrl": "[concat('https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/201-decrypt-running-windows-vm/updateEncryptionSettings-',parameters('volumeType'),'.json')]",	
```

Comment:
Needs to be https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-decrypt-running-windows-vm/updateEncryptionSettings-